Formula/fdroidserver.rb

```diff
@@ -15,25 +15,35 @@ class Fdroidserver < Formula
 
   depends_on :python3
   depends_on "freetype"
-  depends_on "jpeg"
```

Comment:
It hard failed, but maybe there's some flag we can pass to `pillow` in a custom step to override that failure. Will chuck `jpeg` back in either way though.